optimum/habana/accelerate/accelerator.py

```diff
@@ -442,13 +223,15 @@ def prepare_model(self, model: torch.nn.Module, device_placement: bool = None, e
         elif device_placement and not self.verify_device_map(model):
             model = model.to(self.device)
         if not evaluation_mode:
-            if self.distributed_type == GaudiDistributedType.MULTI_HPU and self._distribution_strategy != "fast_ddp":
+            ###############################################################################################################
+            if self.distributed_type == DistributedType.MULTI_HPU and self._distribution_strategy != "fast_ddp":
```

Comment:
```suggestion
            if self.distributed_type == DistributedType.MULTI_HPU and self.distribution_strategy != "fast_ddp":
```

`self._distribution_strategy` does not exist anymore

Reply:
good catch ! thanks